scripts/src/java/org/oppia/android/scripts/proto/filename_pattern_validation_checks.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
```

Comment:
This package doesn't seem correct--ditto for 'model' above. Shouldn't the package be 'proto' and the java_package match the containing folder?

Reply:
Apologies! made this mistake in a hurry.
It's now rectified.